add_npm_package.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Run as ./add_npm_package.sh name version strategy
+program_exists() {
+  which "$@" &> /dev/null;
+}
+
+if !(program_exists crudini); then
+  echo "crudini is not installed - installing..."
+  sudo yum install crudini
```

Comment:
Could you just print the command out instead of running sudo and yum? (Note that Fedora uses `dnf` by default nowadays too.)